src/components/FeedbackSection.tsx

```diff
@@ -60,15 +60,15 @@ const Feedback = ({ result, document, input, secondAttempt }: FeedbackPropsType)
   const background =
     result === 'correct'
       ? correct_background
-      : result === 'incorrect' || !secondAttempt
+      : result === 'incorrect' || result === 'giveUp' || !secondAttempt
       ? incorrect_background
       : hint_background
 
   const message =
     result === 'correct'
       ? 'Toll, weiter so! \nDeine Eingabe ist richtig.'
       : result === 'incorrect' || !secondAttempt
-      ? `Schade, deine Eingabe ist falsch. Die richtige Antwort ist: ${
+      ? `${result === 'incorrect' ? `Schade, deine Eingabe ist falsch.` : ``} Die richtige Antwort ist: ${
           document?.article?.toLowerCase() === ARTICLES.diePlural ? 'die' : document?.article
         } ${document?.word}`
       : `Deine Eingabe ${input} ist fast richtig. Überprüfe Groß- und Kleinschreibung.`
```

Comment:
Maybe this should be split in serveral vars/conditions. Maybe same for above

Reply:
You mean the complete ? : - nesting?

Reply:
Yes, it's kind of confusing imo. But I don't really mind, so you can keep this as well.

Reply:
No, i agree, it is confusing.

Reply:
refactored it.